.gitignore

```diff
@@ -48,4 +48,7 @@ Thumbs.db
 .firebase
```

Comment:
hey, you can just ignore files while pushing. Need not add it in .gitignore and then updating .gitignore.
This might create few problems in the future too.
You can commit only necessary files before making a push. 

Seems like you are committing all the changes
Instead of doing  ```git add .```, commit only the necessary files